cmake/modules/compiler_config.cmake

```diff
@@ -63,6 +64,13 @@ set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-g -O2 -Wall -Wextra -Werror")
 set(CMAKE_C_FLAGS_MINSIZEREL       "-Os -Wall -Wextra -Werror")
 set(CMAKE_CXX_FLAGS_MINSIZEREL     "-Os -Wall -Wextra -Werror")
 
+if(COMPILER_SUPPORTS_IMPLICIT_FALLTHROUGH3)
```

Comment:
The GCC documentation indicates `-Wimplicit-fallthrough=3` is enabled when `-Wextra` is enabled (as it already seems to be on e.g. line 55 and 64). Can you drop this extra check?

Reply:
I tried it without, but it didn't seem to work.  The error message indicated that -Werror=implicit-fallthrough= (without the 3) was active.

Reply:
Whoops. Sorry I missed your PR opening comment, but thanks for double-checking!